### ISSUE TITLE
some go routines call the Get function for one key at the same time, may put pressure on the server

### ISSUE THREAD

**Issue by yiting1122** [goburrow/cache#28]
![image](https://user-images.githubusercontent.com/4957538/126751716-3458fcaf-f966-49ee-9d10-40b1101206d2.png)

if the key is loading ,need wait.

Thanks

**Comment by nqv** [comment]
yes, there is already a TODO https://github.com/goburrow/cache/blob/f6da914dd6e3546dffa8802919dbca80cd33abe3/local.go#L293
